### ISSUE TITLE
Symbol is undefined in IE11

### ISSUE THREAD

**Issue by julkue** [frctl/mandelbrot#56]
![2017-06-14 13_17_31-swissid components _ swissid - internet explorer](https://user-images.githubusercontent.com/8310677/27129819-f527cb7a-5103-11e7-9fdb-2ea56640a289.png)

No emulation, just IE11 with IE11 document mode.

I guess you're using Babel for ES6 transpilation, but not the Babel polyfill.

**Comment by sbaechler** [comment]
@julmot You can always add the Babel polyfill by including the script from the CDN it in the head of your layout template.

https://cdnjs.com/libraries/babel-polyfill

**Comment by julkue** [comment]
Thanks for the tip. However, this is something that Mendelbrot should cover by default.

**Comment by sbaechler** [comment]
The Polyfill is 100kB minified and only needed for IE11. I am against adding it by default.

Maybe using a conditional loader, something like this:
```
    <script>
        if(!('from' in Array)) {
            // IE 11: Load Browser Polyfill
            document.write('<script src="//cdnjs.cloudflare.com/ajax/libs/babel-polyfill/6.23.0/polyfill.min.js" crossorigin="anonymous"></script>');
        }
    </script>
```

**Comment by julkue** [comment]
I won't accept any solution that requires an internet connection, due to the fact that everything else works offline. I'm one of the users working in train where no internet connection should be required.

**Comment by sbaechler** [comment]
The polyfill can also be added to the build folder in the gulp task. The CDN was just an example. IE11 users without internet connection would then still be able to use the guide.

**Comment by tlenex** [comment]
https://github.com/frctl/mandelbrot/pull/46 this PR adds support for IE9+
still waiting to get merged

**Comment by Chapabu** [comment]
Fixed by #82

**Comment by LeBenLeBen** [comment]
Released in 1.2.1
